src/MICore/LaunchOptions.cs

```diff
@@ -209,6 +209,25 @@ public bool ShouldStartServer()
             return !string.IsNullOrWhiteSpace(DebugServer);
         }
 
+        public bool IsValidMiDebuggerPath()
+        {
+            if (!File.Exists(MIDebuggerPath))
+            {
+                return false;
+            }
+            else
+            {
+                // Verify the target is a file and not a directory
+                FileAttributes attr = File.GetAttributes(MIDebuggerPath);
```

Comment:
This came up on one of my recent pull requests - File.Exists will return false if you pass the path of a directory, so you don't need the additional check.